docs/INHIBITOR_LOCKS.md

```diff
@@ -101,7 +101,7 @@ The signal with _False_ is usually the signal on which applications request a ne
 
 Note that watching PrepareForShutdown(true)[?](//secure.freedesktop.org/write/www/ikiwiki.cgi?do=create&from=Software%2Fsystemd%2Finhibit&page=Software%2Fsystemd%2Finhibit%2FPrepareForSleep)/PrepareForSleep(true) without taking a delay lock is racy and should not be done, as any code that an application might want to execute on this signal might not actually finish before the suspend/shutdown cycle is executed.
 
-_Again_: if you watch PrepareForSuspend(true), then you really should have taken a delay lock first. PrepareForShutdown(false) may be subscribed to by applications which want to be notified about system resume events.
```

Comment:
> PrepareForShutdown(false) may be subscribed to by applications which want to be notified about system resume events.

It seems to me that this should be changed to `PrepareForSleep`, instead of the other way around?

Reply:
*sigh*, yeah, it should've been `PrepareForSleep` (and I even read the preceding paragraph)